app/controllers/api/services_controller.rb

```diff
@@ -57,7 +57,7 @@ def create
       flash[:notice] = t('flash.services.create.notice')
       redirect_to admin_service_path(@service)
     else
-      flash.now[:error] = @service.errors.full_messages.to_sentence.presence || I18n.t!('flash.services.create.errors.default')
+      flash.now[:error] = 'Product could not be created'
```

Comment:
Better update and use `I18n.t!('flash.services.create.errors.default')`.

Reply:
updated